tests_e2e/pipeline/pipeline-cleanup.yml

```diff
@@ -1,58 +1,59 @@
 #
 # Pipeline for cleaning up any remaining Resource Groups generated by the Azure.WALinuxAgent pipeline.
 #
-# Deletes any resource groups that are more than a day old and contain string "lisa-WALinuxAgent-"
+# Deletes any resource groups that are older than 'older_than' and match the 'name_pattern' regular expression
 #
-schedules:
```

Comment:
Now the schedule is defined in the UI (we did the same for the main pipeline), so this schedule would be ignored. I'm deleting it altogether.